### DISCUSSION TITLE
API KEY Setting

### DISCUSSION THREAD

**Discussion by kakik0u** [Alex313031/thorium#639] [Q&A] [answered]
In Thorium, Google Sync works even if you are not a member of the Dev Google Group.
However, when I set it up myself, I have to join the DevGoogleGroup to use the sync feature.
Could you please tell me how you are setting up the Google API?
The APIs I have enabled are as follows:
Safe Browsing API 
Cloud Search API 
Admin SDK API
BigQuery API
BigQuery Migration API
BigQuery Storage API
Chrome Sync API
Cloud Datastore API
Cloud Logging API
Cloud Monitoring API
Cloud SQL
Cloud Storage
Cloud Storage API
Cloud Trace API
Cloud Translation API
Geocoding API
Geolocation API
Google Assistant API
Google Calendar API
Google Cloud APIs
Google Cloud Storage JSON API
Google Drive API
Nearby Messages API
Service Management API
Service Usage API
Time Zone API

**Comment by Alex313031** [comment, marked as answer]
Use the keys that thorium uses.

You can use the ./fetch_api_keys.sh in //infra in the repo to get them. They are what Chrome uses. I exclude them from the repo since I don't want to get in trouble.
